tests/unit/utils/sparqlhandler/test_sparqlmanager.py

```diff
@@ -63,9 +63,12 @@ def assert_dsd_component_equal(
     assert component.property_type == property_type.value
     assert component.property_label == property_label
     assert component.csv_col_title == csv_col_title
-    assert component.observation_value_column_titles == observation_value_column_titles
     assert component.required == required
-
+    
+    if observation_value_column_titles is not None and len(observation_value_column_titles) > 0:
+        expected_obs_val_col_titles = [title.strip() for title in observation_value_column_titles.split(',')]
+        actual_obs_val_col_titles = [title.strip() for title in component.observation_value_column_titles.split(',')]
+        assert len(expected_obs_val_col_titles) == len(actual_obs_val_col_titles) and sorted(expected_obs_val_col_titles) == sorted(actual_obs_val_col_titles)
```

Comment:
You could just do a set comparison here tbh, that'd obviate the need for sorting and length checking.